styleguide/media/css/colors.css

```diff
@@ -1,41 +1 @@
-/** color swatch styles */
```

Comment:
We should probably go ahead and delete this file and the reference to it in `styleguide/includes/css.html`